main.py

```diff
@@ -2,6 +2,27 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+from db.data import GENRES, ACTORS
+
 
 def main() -> QuerySet:
-    pass
+
+    for genre in GENRES:
+        Genre.objects.create(name=genre)
+
+    for actor in ACTORS:
+        first_name, last_name = actor
+        Actor.objects.create(first_name=first_name, last_name=last_name)
+
+    Genre.objects.filter(name="Dramma").update(name="Drama")
+    Actor.objects.filter(last_name="Klooney").update(last_name="Clooney")
+    Actor.objects.filter(first_name="Kianu").update(
+        first_name="Keanu",
+        last_name="Reeves"
```

Comment:
what if db will have other Klooneys ?
![image](https://user-images.githubusercontent.com/103201010/224314854-dbe154ff-49a4-4682-aa7b-bb4507cac642.png)
u should make update only for George, the same is for Kianu